test/system/projects/samples_test.rb

```diff
@@ -199,6 +199,31 @@ class SamplesTest < ApplicationSystemTestCase
       end
     end
 
+    test 'should destroy Sample from sample listing page' do
+      visit namespace_project_samples_url(@namespace, @project)
+
```

Comment:
**suggestion:** This test seems a little out of order.  You check what page you are on near the end of the test (lin 220).  I think you should probably ensure that the row count is one less than the count before the delete (maybe by checking the text in the summary under the table (`Display X items`).